import.py

```diff
@@ -649,10 +656,77 @@ def write_to_file(cont: str, filename: str) -> None:
         f.write(cont)
 
 
+def download_decompme(url_str: str) -> None:
+
+    try:
+        # unique decomp.me identifier for scratches
+        parsed_url = urllib.parse.urlparse(url_str)
+        slug = parsed_url.path[1:].split("/")[1]
+    except Exception:
+        print(traceback.format_exc())
+        print(
+            "Failed to parse decomp.me url, it should look like:  https://decomp.me/scratch/aBc12"
```

Comment:
```suggestion
            f"Failed to parse decomp.me url, it should look like:  {DECOMPME_BASE}/scratch/<id>"
```
no reason not to use the constant here. (same in the printout for --decompme)